package-lock.json

```diff
@@ -7,6 +7,7 @@
     "": {
       "name": "weather-overlay",
       "version": "0.0.0",
+      "license": "MIT",
```

Comment:
Please remove this as CJ does not want *any* change to the package-lock.json that is not from themselves, to prevent any kind of hack attempt.

Reply:
Thanks I will try to do that now. Still very new to github. I didn't edit that file either so I am confused on how it got included in this.

Reply:
Just remove the line from your lock, then add, commit and push to your repo, it will automatically add the commit to this pr 😄

Reply:
Thank you. Figured it out!